gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/AuditServiceImpl.java

```diff
@@ -487,7 +487,14 @@ public void createAuditLog(
         try {
             auditRepository.create(audit);
         } catch (TechnicalException e) {
-            LOGGER.error("Error occurs during the creation of an Audit Log {}.", e);
+            LOGGER.error(
+                "Error occurs during the creation of an Audit Log id={}, with organization id {} and environment id {} for user {}.",
+                audit.getId(),
+                audit.getOrganizationId(),
+                audit.getEnvironmentId(),
+                audit.getUser(),
+                e
+            );
```

Comment:
I don't think we should log the user id (PII). The audit id is useless as we are in a creation method. However we may want to log the `referenceType` & `referenceId` 🤔